api/src/pcapi/alembic/versions/20240315T101954_a613c5bc707a_remove_feature_flage_enable_cgr_timeout.py

```diff
@@ -0,0 +1,30 @@
+"""Remove FF ENABLE_CGR_TIMEOUT"""
+
+# pre/post deployment: post
+# revision identifiers, used by Alembic.
+revision = "a613c5bc707a"
+down_revision = "4df45e259f14"
+branch_labels = None
+depends_on = None
+
+
+def get_flag():  # type:ignore[no-untyped-def]
+    from pcapi.models import feature
+
+    return feature.Feature(
+        name="ENABLE_CGR_TIMEOUT",
+        isActive=False,  # feature disable by default
```

Comment:
Il faut ici indiquer la valeur actuellement en prod. Le feature flag y est activé.

```suggestion
        isActive=True,
```